src/lib/extract/extractProps.ts

```diff
@@ -151,6 +151,17 @@ export function extract(instance: Object, props: Object & { style?: [] | {} }) {
   return extractProps(propsAndStyles(props), instance);
 }
 
+export function stringifyPropsForFabric(props: { [key: string]: NumberProp | undefined | null; }): { [key: string]: string; } {
```

Comment:
It looks like this function is beeing called regardless of whether Paper or Fabric is used. Are the props it's generating used only on Fabric or on Paper too?

Reply:
For now the implementation makes the change of all `NumberProp` props to `string` on both archs. I would prefer not to do it but I cannot find a way around it for now. I could add checks for Fabric in all of those places if we don't find another way to handle it.